packages/@coorpacademy-components/src/molecule/table/index.js

```diff
@@ -60,6 +65,28 @@ const Table = (props, context) => {
     );
   });
 
+  const mainClass = classnames(theme === THEMES.COCKPIT ? style.cockpit : style.wrapper);
```

Comment:
comment: imo it's better to have the ternary condition directly on the component using it as it's not used elsewhere, or, defined just before using it (to keep clarity)